### ISSUE TITLE
jsut qeustion not a bug report

### ISSUE THREAD

**Issue by testc2n14** [HansKristian-Work/vkd3d-proton#1992]
yeah sorry for making it a bug report but i didn;t know where else to go to just ask a few question. basically all i want to know is if i can use the to add hardware accelerators mainly ray tracing accelerators on rdna2 gpus to proton games

**Comment by Blisto91** [comment]
This is just a translation layer for running Direct3D 12 games via a Vulkan driver. It doesn't add any extra functionality to the games.

**Comment by testc2n14** [comment]
ahh so i can assume the outputeed vulkan will not use the raytracing hardware on my gpu even tho in windows via d3d 12 it dose

**Comment by mbriar** [comment]
> so i can assume the outputeed vulkan will not use the raytracing hardware on my gpu

it's up to the vulkan driver to use the hardware for ray tracing, and all amd vulkan drivers will do that. There are no extra tools you need to use. However, that doesn't mean performance will always be on-par with windows, but that's not because the hardware is not used.

**Comment by Blisto91** [comment]
If the game supports Ray Tracing on Windows then it will also support Ray Tracing with vkd3d-proton. Thought you were asking if it could add it to games that didn't support it

**Comment by testc2n14** [comment]
no it;s just that with proton it dosen;t use hardware acceleration  so i wanted to see if i could use this to add it to games that support it on windows but not via proton

**Comment by Blisto91** [comment]
Proton does support hardware accelerated RT. vkd3d-proton is a part of Proton.
Which game and Proton version are you having issues with

**Comment by mbriar** [comment]
>  with proton it dosen;t use hardware acceleration

why do you think that?

**Comment by testc2n14** [comment]
becuae in persoan 3 reload in any room with RT preofrmnaces tanks to 5 fps when in windows it would go down the amount you would expect from turning on windows, 40 fps minpu msotly 50-60 fps in roosm with RT. it's a a console port with very minimal pc fetures

**Comment by testc2n14** [comment]
i can;t find a tool on linux to show me the actul utilization of any hardware accelerators  expect the  video decode  and encode

**Comment by mbriar** [comment]
RDNA2 doesn't really have any "hardware accelerators" for RT anyways, just some extra intersection instructions, there is no tool to show it, either on windows or linux. Bad performance also doesn't mean that the hardware isn't used.

**Comment by testc2n14** [comment]
no i jsut looks at the spec sheet for my gpu and it has 60 ray acirators https://www.amd.com/en/products/graphics/amd-radeon-rx-6800

**Comment by mbriar** [comment]
It has exactly has many "ray accelerators" as normal compute units because they are the same thing. Anyways, it's up to the vulkan driver to use the hardware as efficiently as possible, vkd3d-proton already uses the vulkan ray tracing extensions.

**Comment by mbriar** [comment]
Also, the bad performance in Persona 3 might not be because of RT at all but because of: https://gitlab.freedesktop.org/mesa/mesa/-/issues/6260#note_2272979

You can try to run it with `RADV_DEBUG=nongg` as environment variable to see if that helps.
Otherwise you can double check if it is indeed because of RT by just disabling it with `VKD3D_CONFIG=nodxr`.

**Comment by testc2n14** [comment]
should i put thsoe intot he steam propties thing, and dose basic proton that i dwonalod via steam already have this implemented into it. also i can test the preformance with out RT as the game has that as one of it;s 4 graphic settings, (refecltions , render scale, res, and frame rate limit)

**Comment by mbriar** [comment]
Yes, you would add `RADV_DEBUG=nongg %command%` as launch option for the game in steam. Normal Proton has everything.

**Comment by testc2n14** [comment]
well ty for all ur help

**Comment by mbriar** [comment]
Did RADV_DEBUG=nongg change anything?

**Comment by testc2n14** [comment]
no it did not

**Comment by pixelcluster** [comment]
> RDNA2 doesn't really have any "hardware accelerators" for RT anyways, just some extra intersection instructions

I don't see why these intersection instructions wouldn't count as hardware accelerators?

@testc2n14: Which Mesa version are you using? If you have `vulkaninfo` installed, you can check this by looking at the output of `vulkaninfo --summary`. There have been significant performance improvements for raytracing UE4 games like Persona 3 Reload in Mesa 24.0, perhaps upgrading to that version will help.
